docs/pages/management/operations/self-hosted-automatic-agent-updates.mdx

```diff
@@ -0,0 +1,117 @@
+---
+title: Setting up self-hosted automatic agent updates
+description: How to setup automatic agent update for self-hosted Teleport
+---
+
+<Details
+  title="Version warning"
+  opened={true}
+  scope={["enterprise"]}
+  scopeOnly={true}
+  min="13.0"
+>
+  Automatic agent update is available starting from Teleport `13.0`.
+</Details>
+
+Starting with version 13, Teleport supports automatic agent updates for
+systemd-based Linux distributions using `apt` or `yum` package managers,
+and Kubernetes clusters. The [automatic updates architecture 
+page](../../architecture/automatic-agent-update.mdx) describes how agent
+updating works.
+
+This guide covers how to set up the automatic update infrastructure. If this is
+already done, or you are a Teleport Cloud user, you can directly
+[enroll agents into automatic updates](./enroll-agent-into-automatic-updates.mdx).
+
+## Requirements
+
+- Self-hosted Teleport cluster running.
+- `tctl` admin rights.
+- Either:
+  - a public S3/GCS bucket,
+  - a web server accessible from all agents with valid TLS certificates.
+
+## Step 1/3. Create release channel files
+
+A release channel contains two pieces of information: the targeted version
+and if the update is critical. Updaters subscribe to a release channel and will
+update to the provided version during a maintenance window if possible. If the
+update is critical, updaters will ignore the maintenance schedule and update as
+soon as possible.
+
+Create a directory for the new release channel `current`.
+
+```code
+$ mkdir current/
+```
+
+Make the `current` release channel target the version (=teleport.version=):
+
+```code
+$ echo -n "(=teleport.version=)" > current/version
+```
+
+And mark the update as not critical:
+
+```code
+$ echo -n "no" > current/critical
+```
+
+## Step 2/3. Host the release channel files on the version server
+
+The release channel must be hosted on a webserver with trusted TLS certificates
+and reachable by all agents.
+
+Public cloud buckets like Amazon S3 or Google Cloud Storage are good
```

Comment:
WE should provide examples of using S3 and Google Cloud storage.

Reply:
When we'll come back to automatic updates we'll likely have those endpoints served by the proxies themselves.